ios/Rainbow.xcodeproj/project.pbxproj

```diff
@@ -431,8 +431,8 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 13B07F931A680F5B00A75B9A /* Build configuration list for PBXNativeTarget "Rainbow" */;
 			buildPhases = (
-				66A29CC9251106EE00481F4A /* Disable Reanimated if needed */,
 				B2BF48CC79AEFAAAA7A76369 /* [CP] Check Pods Manifest.lock */,
```

Comment:
hey could u actually remove all the changes to this file 🙏 i’m not sure why it would have changed, but regardless this particular change has nothing to do with the change to ‘package.json’ so we shouldn’t include it